src/main/java/de/uhd/ifi/se/decision/management/jira/filtering/FilterSettings.java

```diff
@@ -86,6 +95,18 @@ public FilterSettings(@JsonProperty("projectKey") String projectKey,
 		this.isHierarchical = false;
 		this.isIrrelevantTextShown = false;
 		this.createTransitiveLinks = false;
+		this.setIrrelevantTextShown(false);
+
+		this.linkImpact = new HashMap<>();
+		DecisionKnowledgeProject.getAllNamesOfLinkTypes().forEach(entry -> {
+			linkImpact.put(entry, 1.0f);
+		});
+		this.displayType = "";
```

Comment:
Please consider having one CiaSettings attribut instead of the single parameters in the FilterSettings. This would reduce the amount of attributes in this class.